sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -163,6 +324,39 @@ export interface ServiceBusClientOptions {
     webSocketOptions?: WebSocketOptions;
 }
 
+// @public
+export class ServiceBusManagementClient extends ServiceClient {
+    constructor(connectionString: string, options?: ServiceBusManagementClientOptions);
+    createQueue(queueName: string): Promise<CreateQueueResponse>;
+    createQueue(queue: QueueDescription): Promise<CreateQueueResponse>;
+    createRule(topicName: string, subscriptionName: string, rule: RuleDescription): Promise<CreateRuleResponse>;
+    createSubscription(topicName: string, subscriptionName: string): Promise<CreateSubscriptionResponse>;
+    createSubscription(subscription: SubscriptionDescription): Promise<CreateSubscriptionResponse>;
+    createTopic(topicName: string): Promise<CreateTopicResponse>;
+    createTopic(topicOptions: TopicDescription): Promise<CreateTopicResponse>;
```

Comment:
topicOptions -> topic

Reply:
Done

Reply:
I see the code changed, but the api.md file still has the old name

Reply:
Now its updated :)